README.md

```diff
@@ -44,13 +44,15 @@ If your node system of choice is missing, feel free to integrate it and let us k
 <th style="text-align:center"><img src="docs/assets/raspiblitz-dark.svg#gh-light-mode-only" height="50px" alt="RaspiBlitz Logo" /><img src="docs/assets/raspiblitz-light.svg#gh-dark-mode-only" width="45px" alt="RaspiBlitz Logo" /></th>
 <th style="text-align:center"><img src="docs/assets/umbrel.svg" width="45px" alt="Umbrel Logo" /></th>
 <th style="text-align:center"><img src="docs/assets/citadel.png" width="45px" alt="Citadel Logo" /></th>
+<th style="text-align:center"><img src="docs/assets/mynode.png" width="45px" alt="MyNode Logo" /></th>
 </tr>
 </thead>
 <tbody>
 <tr>
 <td style="text-align:center"><a href="https://github.com/rootzoll/raspiblitz">RaspiBlitz</a>: v1.7.2*</td>
 <td style="text-align:center"><a href="https://getumbrel.com">Umbrel</a>: v0.4.15</td>
 <td style="text-align:center"><a href="https://runcitadel.space">Citadel</a>: v0.0.1</td>
+<td style="text-align:center"><a href="https://mynodebtc.com">MyNode</a>: v0.3.5</td>
```

Comment:
I suppose it is [v0.3.05](https://github.com/mynodebtc/mynode/releases/tag/v0.3.05)?

```suggestion
<td style="text-align:center"><a href="https://mynodebtc.com">MyNode</a>: v0.3.05</td>
```

Reply:
Thanks, that's right! I updated it.